google_tenant_project_bootstrap/main.tf

```
@@ -0,0 +1,26 @@
+module "google_gke_tenant" {
+  source = "github.com/mozilla/terraform-modules//google_gke_tenant?ref=main"
+
+  project_id         = var.project
+  environment        = var.environment
+  cluster_project_id = var.gke_cluster_project_id
+  namespace          = var.application
```

```suggestion
  application        = var.application
```
A namespace is a combination of $application-$env, so I think we just need to adjust this here. I did a bad job of reviewing https://github.com/mozilla/terraform-modules/blob/main/google_gke_tenant/variables.tf#L19-L24 so it should be adjusted there as well.